cocotb/share/makefiles/Makefile.inc

```diff
@@ -93,12 +93,21 @@ COCOTB_HDL_TIMEPRECISION ?= 1ps
 
 export COCOTB_RESULTS_FILE
 
-# Depend on all Python from the cocotb package. This triggers a
-# recompilation of the simulation if cocotb is updated.
-CUSTOM_SIM_DEPS += $(shell find $(COCOTB_PY_DIR)/cocotb/ -name "*.py")
+ifeq ($(OS), Msys)
```

Comment:
This is the only value for OS we care about on Windows? What's the value of OS when in an Anaconda environment, or a MINGW environment?

Reply:
it works on Windows and Miniconda. didnt check on MINGW